docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+services:
+  todoapp:
+    image: todoapp:1.0.0
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+      - "8080:8080"
+    networks:
+      - todoapp-network
+    depends_on:
+      - mysql-db
+    restart: unless-stopped
+
+  mysql-db:
+    image: mysql-local:1.0.0
+    build: 
+      context: .
+      dockerfile: Dockerfile.mysql
```

Comment:
Ensure that the `Dockerfile.mysql` exists in the context directory and is correctly configured to build the MySQL image.